NICE.Identity.Management/ClientApp/src/views/UsersList/__tests__/UsersList.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { mount, shallow } from "enzyme";
+import fetchMock from "fetch-mock";
+import toJson from "enzyme-to-json";
+import { UsersList } from "../UsersList";
+import users from "./users.json";
+import { MemoryRouter } from "react-router";
+import { nextTick } from "../../../utils/nextTick";
+
+afterEach(fetchMock.reset);
+
+it("should show loading message before data has been loaded", () => {
```

Comment:
If we're doing per-component error handling then we should have a test here for error states (both 500 and 401/403 I guess)